luda-editor/luda-editor-client/src/app/editor/sequence_player/mod.rs

```diff
@@ -172,10 +173,13 @@ impl SequencePlayer {
                     language: props.language,
                     subtitle_play_duration_measurer: &props.subtitle_play_duration_measurer,
                 }),
-                render_buttons(&ButtonsProps {
-                    xywh: &buttons_xywh,
-                    playback_status: &playback_status,
-                }),
+                match props.with_buttons {
+                    true => render_buttons(&ButtonsProps {
+                        xywh: &buttons_xywh,
+                        playback_status: &playback_status,
+                    }),
+                    false => RenderingTree::Empty,
+                },
```

Comment:
Make buttons optional.

Reply:
I believe you should update `player_screen_xywh`'s height if there are no buttons. please check it.

Reply:
Oh! I missed it. Thank you.
Fixed with 859bd4c